test/quadquizaminos_web/live/power_up_test.exs

```diff
@@ -176,10 +176,10 @@ defmodule QuadquizaminosWeb.PowerUpTest do
       render_click(view, "transform_block", %{"x" => "5", "y" => "20", "color" => "purple"})
       html = render_click(view, "add_block", %{"x" => "7", "y" => "18"})
 
-      refute html =~
+      assert html =~
```

Comment:
You have interchanged the assertion of the test.  That means the test is not correct/well implemented, if it's failing kindly follow it keenly to understand where the loophole might be coming from